+sw_tests/+system_tests/systemtest_spinwave_symbolic_nips.m

```diff
@@ -0,0 +1,117 @@
+classdef systemtest_spinwave_symbolic_nips < sw_tests.system_tests.systemtest_spinwave
+
+    properties
+        reference_data_file = '';
+        swobj_nn = [];
+        symspec = [];
+    end
+
+    properties (TestParameter)
+        test_spectra_function_name = {'sw_neutron', 'sw_egrid', 'sw_instrument', ...
+                                      'sw_omegasum', 'sw_plotspec', 'sw_tofres'};
+    end
+
+    methods (TestClassSetup)
+        function prepareForRun(testCase)
+            % Skips all these tests if the symbolic toolbox is not installed
+            testCase.assumeTrue(sw_hassymtoolbox(), 'Symbolic Toolbox not installed');
+            % Symbolic calculation, based on "Magnetic dynamics of NiPS3", A.R.Wildes et al., Phys. Rev. B in press
+            nips = spinw();
+            nips.genlattice('lat_const', [5.812, 10.222, 6.658], 'angled', [90, 107.16, 90], 'sym', 12);
+            nips.addatom('r', [0 0.333 0], 'S', 1, 'label', 'MNi2');
+            nips.gencoupling();
+            nips.addmatrix('label', 'J1', 'mat', 1);
+            nips.addcoupling('mat', 'J1', 'bond', 1);
+            nips.addcoupling('mat', 'J1', 'bond', 2);
+            nips.genmagstr('mode', 'direct', 'k', [0 1 0], 'S', [1 0 0; -1 0 0; -1 0 0; 1 0 0]');
+            % The full system is too complicated to determine the general dispersion 
+            % It will run for several hours and then run out of memory.
+            % Instead for some tests we simplify it to be able to run through the full calculation
```

Comment:
Might be worth a comment saying in what way it was simplified?

Does using the NiPS3 system give us any advantages over using a simpler system for these tests?